.luarc.json

```diff
@@ -1,8 +1,14 @@
 {
+  "$schema": "https://raw.githubusercontent.com/LuaLS/vscode-lua/master/setting/schema.json",
   "workspace.library": ["./lua/types"],
-  "runtime.version": "Lua 5.1",
-  "hint.enable": true,
-  "completion.autoRequire": true,
-  "completion.callSnippet": "Replace",
-  "completion.displayContext": 8
+  "runtime.version": "Lua 5.4",
+  "hint": {
+    "enable": true,
+    "setType": true
+  },
+  "completion": {
+    "autoRequire": true,
```

Comment:
Do we _REALLY_ need `autoRequire = true`? If so, how come?

I always disable it in my LSP settings so I'm clueless.